src/main/java/address/browser/BrowserManager.java

```diff
@@ -1,53 +1,50 @@
 package address.browser;
 
-import com.teamdev.jxbrowser.chromium.Browser;
-import com.teamdev.jxbrowser.chromium.EditorCommand;
-import com.teamdev.jxbrowser.chromium.dom.By;
-import com.teamdev.jxbrowser.chromium.dom.DOMElement;
-import com.teamdev.jxbrowser.chromium.dom.events.DOMEventType;
-import com.teamdev.jxbrowser.chromium.events.FinishLoadingEvent;
-import com.teamdev.jxbrowser.chromium.events.LoadAdapter;
+import address.model.ModelPerson;
+import address.model.Person;
+
+import javafx.collections.ObservableList;
+import javafx.scene.control.TabPane;
 
 /**
  * Manages the browser.
  */
 public class BrowserManager {
 
-    private Browser browser;
 
-    public BrowserManager() {
-        this.browser = new Browser();
-        registerListeners();
-    }
+    public static final int NUMBER_OF_PRELOADED_PAGE = 3;
+
+    private AddressBookBrowser browser;
 
-    private void registerListeners() {
-        this.browser.addLoadListener(new LoadAdapter() {
-            @Override
-            public void onFinishLoadingFrame(FinishLoadingEvent finishLoadingEvent) {
-                automateClickingAndScrolling();
-            }
-        });
+    private ObservableList<ModelPerson> filteredModelPersons;
+
+    public BrowserManager(ObservableList<ModelPerson> filteredModelPersons) {
+        this.filteredModelPersons = filteredModelPersons;
+        browser = new AddressBookBrowser(NUMBER_OF_PRELOADED_PAGE, this.filteredModelPersons);
+        browser.registerListeners();
     }
 
     /**
-     * Frees resources allocated to the browser.
+     * Loads the person's profile page to the browser.
+     * PreCondition: filteredModelPersons more than one person.
```

Comment:
Should be >= 1. That's a mistake. I have changed it to follow the same convention as browser